From: Simon McVittie <smcv@debian.org>
Date: Sun, 22 Sep 2024 13:46:49 +0100
Subject: testsuite/tools/validate: Use softpipe driver for validate test

The llvmpipe driver sometimes spams stderr with messages from libLLVM,
which cause this particular test to fail.

Bug: https://gitlab.freedesktop.org/mesa/mesa/-/issues/11870
Forwarded: no
---
 testsuite/tools/validate | 1 +
 1 file changed, 1 insertion(+)

diff --git a/testsuite/tools/validate b/testsuite/tools/validate
index ed9adb6..ae0eb80 100755
--- a/testsuite/tools/validate
+++ b/testsuite/tools/validate
@@ -6,6 +6,7 @@ TEST_RESULT_DIR=${TEST_RESULT_DIR:-/tmp}/validate
 
 # https://gitlab.freedesktop.org/mesa/mesa/-/issues/10293
 export LIBGL_ALWAYS_SOFTWARE=true
+export GALLIUM_DRIVER=softpipe
 
 mkdir -p "$TEST_RESULT_DIR"
 
